content/en/platform/corda/5.2/deploying-operating/deployment/deploying/_index.md

```diff
@@ -1087,6 +1088,18 @@ commonPodLabels:
   sidecar.istio.io/inject: !!str true # explicitly enable Istio integration for all Corda pods
 ```
 
+### Ledger Repair
+
+By default, the [Corda ledger repair functionality]({{< relref "../../../developing-applications/ledger/ledger-repair.md" >}}) runs every ten minutes. You can modify this schedule by setting the `net.corda.ledger.utxo.repair.schedulePeriod` system property for each database worker. This property is set in seconds. For example, to modify the schedule to every 15 minutes:
+
+```yaml
+workers:
+  db:
+    javaOptions: "-Dnet.corda.ledger.utxo.repair.schedulePeriod=900"
```

Comment:
As we have `-XX:MaxRAMPercentage=75` as the default value for this field, we should probably include it in the example; otherwise, they'll be unsetting it.
```suggestion
    javaOptions: "-XX:MaxRAMPercentage=75 -Dnet.corda.ledger.utxo.repair.schedulePeriod=900"
```